src/inputs/internal/SelectFieldInput.tsx

```diff
@@ -73,7 +66,6 @@ export function SelectFieldInput<O, V extends Value>(props: SelectFieldInputProp
   const readOnlyStyles = isReadOnly ? Css.bn.pl0.pt0.add("backgroundColor", "unset").$ : {};
   const tid = useTestIds(inputProps); // data-testid comes in through here
   const isMultiSelect = state.selectionManager.selectionMode === "multiple";
-  const [isSizeCurrentlyBasedOnContent, setIsSizeBasedOnContent] = useState(true);
```

Comment:
This boolean `useState` was just being set true/false in focus/blur, and we already have an `isFocused` hanging around, so used that instead.